lib/matplotlib/tests/test_image.py

```diff
@@ -72,11 +72,12 @@ def test_interp_nearest_vs_none():
     ax2.set_title('interpolation nearest')
 
 
-@image_comparison(baseline_images=['figimage-0', 'figimage-1'], extensions=['png'])
-def test_figimage():
+@image_comparison(baseline_images=['figimage-0'],
+                  extensions=['png','pdf'])
+def test_figimage0():
     'test the figimage method'
 
-    for suppressComposite in False, True:
+    for suppressComposite in [False]:
```

Comment:
I am a bit concerned that we are losing testing the `supressComposite=True` code path.

Reply:
Oh, never mind, I should have read the next part of diff :sheep:

Reply:
Yeah, maybe I should clean this a bit. The for-loop worked when the test only did `png`, but never got to the second pdf for some reason.  This (ahem) "fixed" the problem.